compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -84,6 +84,7 @@ pub(crate) fn optimize_into_acir(
     // This pass must come immediately following `mem2reg` as the succeeding passes
     // may create an SSA which inlining fails to handle.
     .run_pass(Ssa::inline_functions_with_no_predicates, "After Inlining:")
+    .run_pass(Ssa::array_get_optimization, "After Array Get Optimizations:")
```

Comment:
💭 This name is temporary, we should find a name that better describes this pass.